frog/imports/api/activities.js

```diff
@@ -93,11 +93,12 @@ export const findOneActivityMongo = (id: string) => {
 export const addActivity = (
   activityType?: string,
   data: ?Object = {},
-  id: string,
+  id?: string,
   configVersion: ?number,
   groupingKey: ?string,
   parentId: ?string
 ) => {
```

Comment:
All fields of `addActivity` are Maybe type. If it is within the scope of this PR, we could identify some fields that are required to avoid a potential undefined bug.